### PR TITLE
pacific: cephadm backport February (2)

### PR DESCRIPTION
**Pull request by jmolmo**
Backport of #38549, #38883, #39058, #39355, #38505, #39364, #39381, #39384, #39385, #39398, #39384, #39404, #39430, #38881, #39458, #39473, #39482, #39424, #39490, #39323, #39467, #39464, #39496, #38897,  #39153, #39266

Useful information for the next backport:
Last cephadm commit merged from master in this backport:
#39496 9df5a6d73ed Tue, 16 Feb 2021 11:07:08 -0600 Tue, 16 Feb 2021 11:07:08 -0600 mgr/cephadm: fix host refresh

conflicts: 
#38549 (src/pybind/mgr/cephadm/serve.py, src/pybind/mgr/cephadm/services/nfs.py, src/pybind/mgr/cephadm/tests/test_services.py)
#38883 (src/pybind/mgr/cephadm/inventory.py, src/pybind/mgr/cephadm/module.py, src/pybind/mgr/cephadm/tests/test_cephadm.py
#39058 (src/pybind/mgr/cephadm/tests/test_upgrade.py)
#38505 (doc/dev/cephadm/index.rst)
#38881 (src/cephadm/cephadm)
#39153: src/pybind/mgr/cephadm/services/nfs.py, src/pybind/mgr/tox.ini, src/pybind/mgr/cephadm/tests/test_cephadm.py
#39266: src/pybind/mgr/cephadm/serve.py
#38897: src/pybind/mgr/cephadm/tests/test_upgrade.py

### CODE REVIEW

File: src/cephadm/cephadm [hunk 1]
```diff
@@ -1,7 +1,14 @@
 #!/usr/bin/python3
 
-DEFAULT_IMAGE = 'docker.io/ceph/daemon-base:latest-pacific-devel'
-DEFAULT_IMAGE_IS_MASTER = False
+# Default container images -----------------------------------------------------
+DEFAULT_IMAGE = 'docker.io/ceph/daemon-base:latest-master-devel'
```
Comment (sebastian-philipp) [inline src/cephadm/cephadm:4]: DEFAULT_IMAGE = 'docker.io/ceph/daemon-base:latest-pacific-devel'

**Review by sebastian-philipp — CHANGES_REQUESTED** [review]

File: src/cephadm/cephadm (hunk 1 above)
Comment (jmolmo) [reply src/cephadm/cephadm]: fixed

**Comment by sebastian-philipp** [comment]
I still don't get why you have so many conflicts

**Review by ktdreyer — APPROVED** [review]
I've tested this on RHEL 8.

**Comment by sebastian-philipp** [comment]
too many conflicts. replaced by #39623

**Pull request closed without merging**